nuxt.config.ts

```diff
@@ -47,7 +47,9 @@ export default defineNuxtConfig({
     'floating-vue/dist/style.css',
     '~/styles/default-theme.css',
     '~/styles/vars.css',
-    '~/styles/global.css',
+    ...process.env.TAURI_PLATFORM === 'macos'
+      ? ['~/styles/global.css']
+      : ['~/styles/global.css', '~/styles/scrollbars.css'],
```

Comment:
We could keep global out, no?
```suggestion
    '~/styles/global.css',
    ...process.env.TAURI_PLATFORM === 'macos'
      ? []
      : ['~/styles/scrollbars.css'],
```

Reply:
Looks neat!